Finjector.Web/ClientApp/src/types.ts

```diff
@@ -1,4 +1,4 @@
-export interface Chart {
+export interface Coa {
```

Comment:
I don't have a great suggestion but don't love the `Coa` here, plus we use it everywhere.  Maybe something like `ChartDetail` or something?  I know we already have `ChartData` so it would be similar

Reply:
i don't either, this is just reflecting Coa.cs. i'll change everything everywhere all at once, just wanted to distinguish this from Chart.cs